kernel/core/extend/userland.c

```diff
@@ -3,7 +3,7 @@
 #include <extend/userland.h>
 #include <extend/intr.h>
 
-void __attribute__((section(".usr"),aligned(4))) userland()
+void __attribute__((section(".task2"),aligned(4))) userland()
```

Comment:
Why change it ? Leave it in usr if you don't need it, unless for testing I guess.

Reply:
Bro is reacting to what he has done in his core_polish that I merged...